homeassistant/components/stookalert/binary_sensor.py

```diff
@@ -0,0 +1,76 @@
+"""This component provides support for Stookalert Binary Sensor."""
+import logging
+import voluptuous as vol
+import stookalert
+from datetime import timedelta
+
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.entity import Entity
+from homeassistant.components.binary_sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_NAME, STATE_OFF, STATE_ON
```

Comment:
Please keep the import grouped and sorted -> https://developers.home-assistant.io/docs/en/development_guidelines.html#ordering-of-imports